dev_guide/deployments.adoc

```diff
@@ -191,6 +191,53 @@ To see what the rollback would look like without performing the rollback:
 $ oc rollback <deployment_config> --dry-run
 ----
 
+[[executing-commands-inside-a-container-deployments]]
+== Executing Commands Inside a Container
```

Comment:
Yes, this is something I've been doing in order to prevent conflicts. Not sure if I've discussed it with anyone... It's probably not necessary in this case, but I'm nothing if not consistent :P